src/modules/selectors/index.js

```diff
@@ -0,0 +1,7 @@
+export const getListFilms = state => {
```

Comment:
now we can use it together with `useSelector()`

Reply:
done